File: classes/parte-4-control-del-programa/062-coincidencia-de-patrones-match-when/implementaciones/typescript/main.ts
import { readFileSync } from "node:fs";

const n: number = parseInt(readFileSync(0, "utf8").trim(), 10);
const signo: string = n > 0 ? "positivo" : n < 0 ? "negativo" : "cero";
console.log(`signo=${signo}`);

File: classes/parte-4-control-del-programa/062-coincidencia-de-patrones-match-when/implementaciones/c/main.c
#include <stdio.h>

int main(void) {
    long n;
    if (scanf("%ld", &n) != 1) return 1;
    const char *signo = n > 0 ? "positivo" : (n < 0 ? "negativo" : "cero");
    printf("signo=%s\n", signo);
    return 0;
}
